index.rst

```diff
@@ -13,13 +13,93 @@
 Abstract
 ========
 
-This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
+This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. 
+The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
 
-Add content here
-================
+Guidelines for Operations
+=========================
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Survey mode is intended to be the primary mode for Auxiliary Telescope (AuxTel) operations. 
+It is a passive mode of operations, with nighttime observations driven by the Scheduler with minimal staff intervention. 
+The central tenant of operations in this mode is that the AuxTel operations are secondary to any responsibilities the summit staff may have related to Simonyi Survey Telescope operations. 
+
+This results in the following guidelines for operations in survey mode: 
+
+- The observing staff are not be required to actively monitor or report on incoming data quality.
+- Logging will be minimal; limited to short narrative statements needed to report on changes during the night or provide context for faults and recovery attempts.
+- Faults will be recovered when observing staff have available time, and the AuxTel may remain in Standby state during the night for extended periods of time. 
+- Remote support will be available for fault resolution on an on-call basis. 
+- Any faults which put hardware at risk will be result in closing the AuxTel early for the night.
+- The decision for closing the AuxTel early is to be made by observing staff on summit, no additional approval is required.
+
+Daytime activities
+==================
+
+Daytime Checkouts
+-----------------
+Daytime checkouts should be run once per week, preferable on Monday afternoons. 
```

Comment:
When the checkouts/startups fail, they catch our attention regardless. I am just having trouble resolving the 99% hands-off model and just naturally getting sucked into troubleshooting. Without the daily mechanical checkout, even without calibrations, we are more likely to be overloaded at the start of the night and that is a worse time to be notified of technical errors. Twilight is typically our busiest time of night.

Reply:
That's true, especially on nights where we can't get to calibrations/venting before dinner. The optimist in me hopes that as we start to operate more often and with fewer changes we move to a more reliable system, and most nights start off like the second or third night of a run. 

The realist thinks we'll end up with some amount (maybe a large amount) of nights where we don't open at all because we never get past the the start of the night. 

I'm okay with both, but in any case we have to train people not to get distracted by AuxTel if they can help it.

Reply:
All right. It's still a bit of a conflict with me to think about a telescope that we just suffer time loss from, but I can work on that from my side if there's some kind of revision in 6 months that adapts the model to time loss vs. investment.